cmd/daemon/player.go

```diff
@@ -314,6 +316,39 @@ func (p *AppPlayer) handleDealerRequest(req dealer.Request) error {
 
 func (p *AppPlayer) handleApiRequest(req ApiRequest) (any, error) {
 	switch req.Type {
+	case WebApiRequestType:
+		resp, err := p.sess.WebApi(req.WebAPI.Method, req.WebAPI.Path, req.WebAPI.Query, nil, nil)
+		defer resp.Body.Close()
+		if err != nil {
+			return nil, fmt.Errorf("failed to send web api request")
+		}
+
+		// this is the status we want to return to client not just 500
+		switch resp.StatusCode {
+		case 404:
+			return nil, ErrNotFound
+		case 405:
+			return nil, ErrMethodNotAllowed
+		}
+
+		// check for content type if not application/json
+		if !strings.Contains(resp.Header.Get("content-type"), "application/json") {
+			respString, err := io.ReadAll(resp.Body)
+			if err != nil {
+				return nil, fmt.Errorf("failed to read response body")
+			}
+			// return as plain text
+			return string(respString), nil
```

Comment:
This will be JSON encoded by `handleRequest` turning it into `"..."`, is it actually possible for the API to not return JSON?

Reply:
for some reason this works and output "ABC..."